index.php

```diff
@@ -67,6 +121,44 @@ function onClickArrow() {
       </div>
     </section>
 
+        <section class="row section d-flex align-items-center justify-content-around rounded">
+
+          <div class="col-lg-7 d-flex justify-content-center rounded model-backround p-2" data-bs-toggle="tooltip" title="<?php echo _('Modelled by ppemawm'); ?>">
+            <div class="animated-sprite ratio ratio-4x3"
+                data-frame-count="39"
+                data-frames-per-row="12"
+                style="background-image: url('images/f16-main-landing-gear-spritesheet.avif')">
+            </div>
+          </div>
+
+          <div class="col-lg-4 text-light text-center text-lg-start px-md-4 rounded text-backround">
+            <h3 class="section-title mt-3"><?php echo _('For All Professional Needs...'); ?></h3>
+            <p class="section-body whitelinks">
+              <?php echo _('Looking for something that contains everything you need in an industrial environment? FreeCAD is the answer. After being available for "real work" for a very long time, it finally has become official, publishing version 1.0. Stop worrying about license costs and start doing all your professional work as a real owner.'); ?>
+            </p>
```

Comment:
I think this is overselling what we offer, how about the following instead?

```suggestion
            <h3 class="section-title mt-3"><?php echo _('For Professional Design Needs...'); ?></h3>
            <p class="section-body whitelinks">
              <?php echo _("FreeCAD is a powerful and evolving open-source parametric 3D modeler, well-suited for many professional and technical tasks. With the release of version 1.0, it marks a new milestone in stability and usability. FreeCAD puts you in control - no licensing fees, no vendor lock-in. It's not just cost-effective - it’s yours to adapt and build upon."); ?>
            </p>
```

Reply:
I understand the suggestion, but what do you mean by overselling, I think everything is pretty neat.

Reply:
I mean the text promised more than freecad can deliver - Freecad doesn't contain everything you need in an industrial environment.

Reply:
Of course, it doesn't, which program can do this anyway. Just like Red Bull won't give wings. There is a literary art here.